paper-input-behavior.html

```diff
@@ -335,6 +346,50 @@
     },
 
     /**
+     * Forward focus to inputElement
+     */
+    _onFocus: function() {
+      if (!this._shiftTabPressed) {
+        // iron-autogrow-textarea
+        if (this.inputElement.textarea) {
+          this.inputElement.textarea.focus();
+        } else {
+          this.inputElement.focus();
+        }
+      }
+    },
+
+    /**
+     * Forward blur to inputElement
+     */
+    _onBlur: function() {
+      if (!this._shiftTabPressed) {
+        // iron-autogrow-textarea
+        if (this.inputElement.textarea) {
+          this.inputElement.textarea.blur();
+        } else {
+          this.inputElement.blur();
+        }
+      }
+    },
+
+    /**
+     * Handler that is called when a shift+tab keypress is detected by the menu.
+     *
+     * @param {CustomEvent} event A key combination event.
+     */
+    _onShiftTabDown: function(event) {
+      var oldTabIndex;
+      this._shiftTabPressed = true;
+      oldTabIndex = this.getAttribute('tabindex');
+      this.setAttribute('tabindex', '-1');
+      this.async(function() {
+        this.setAttribute('tabindex', oldTabIndex);
+        this._shiftTabPressed = false;
+      // NOTE(cdata): polymer/polymer#1305
```

Comment:
(technically you made this note, not me.. unless you shamelessly copy-pasted this, which would never happen, right? ;)